book/website/reproducible-research/case-studies/statistical-methods-manuscript.md

```diff
@@ -0,0 +1,146 @@
+
+that point to the state of the repository at a specific point in time (via
+th git tagging feature). 
+This means tha the links will remain valid and unchanged even if there are 
+later corrections to the contents of the repository!
+Binder supports multiple user interfaces. 
+This is leveraged to provide and Jupyter lab Integrated Development Environment
+view on the repository to explore file, the notebook, or to open a shell for
+further commands. 
+The second badge, directly opens an interactive Shiny app that illustrates 
+some of the points discussed in the manuscript and requires no familiarity with
+programming at all.
+
+
+### Workflow management via snakemake
+
+Since Jupyterlab also allows to open a shell in the repository instance opended
+via a binder link, 
+another feature of the repository can be used to reproduce the *entire manuscript from scratch*.
+The python workflow manager [snakemake](https://snakemake.readthedocs.io/en/stable/)
+was used to define all required steps in a `Snakefile`.
+To execute this workflow,
+simply open a shell in the online version of Jupyterhub (via the binderlink)
+and type
+```
+snakemake -F --cores 1  manuscript
+```
+This will execute all required steps in turn, i.e., first create all plots 
+and then compiling the actual `latex/main.pdf` file from the LaTeX sources.
+All relevant configurations for binder ate located in `.binder`.
```

Comment:
```suggestion
Since JupyterLab also allows to open a shell in the repository instance opened
via a binder link, 
the repository can be used to reproduce the *entire manuscript from scratch*.
The python workflow manager [snakemake](https://snakemake.readthedocs.io/en/stable/)
was used to define all required steps in a `Snakefile`.
To execute this workflow,
you can open a shell in the online version of Jupyterhub (via the binder link)
and type the following command:
```
snakemake -F --cores 1  manuscript
```
This will execute all the required steps, that will first create all plots 
and then compile the actual `latex/main.pdf` file from the LaTeX sources.
All relevant configurations for the binder are located in `.binder` folder.
```